ios/RNFirebase/notifications/RNFirebaseNotifications.m

```diff
@@ -100,10 +102,17 @@ - (void)didReceiveLocalNotification:(nonnull UILocalNotification *)localNotifica
 
 RCT_EXPORT_METHOD(complete:(NSString*)handlerKey fetchResult:(UIBackgroundFetchResult)fetchResult) {
     if (handlerKey != nil) {
-        void (^completionHandler)(UIBackgroundFetchResult) = completionHandlers[handlerKey];
-        if(completionHandler != nil) {
-            completionHandlers[handlerKey] = nil;
-            completionHandler(fetchResult);
+        void (^fetchCompletionHandler)(UIBackgroundFetchResult) = fetchCompletionHandlers[handlerKey];
+        if (fetchCompletionHandler != nil) {
+            fetchCompletionHandlers[handlerKey] = nil;
+            fetchCompletionHandler(fetchResult);
+        }
+        else {
+            void(^completionHandler)(void) = completionHandlers[handlerKey];
+            if (completionHandler != nil) {
+                completionHandlers[handlerKey] = nil;
+                completionHandler();
```

Comment:
As above; any reason for removing the previous behaviour of `completionHandler(fetchResult);`? Is this not going to break the previous behaviour?

Reply:
Same  as above the NSUserNotificationCenter completion handler doesn't take any arguments in its signature so you just need to call it.